app/models/coalition.rb

```diff
@@ -13,4 +13,6 @@
 #
 class Coalition < ApplicationRecord
   validates :name, uniqueness: true
+  # A coalition has many Organizations. We use the VitaPartner.organizations scope to ensure we only find Organizations, not Sites.
+  has_many :organizations, -> { organizations }, class_name: "VitaPartner", foreign_key: "coalition_id"
```

Comment:
If I'm understanding this right, these are organizations because only organizations store the coalition_id, so something having a coalition_id means that it is an organization.